### ISSUE TITLE
apps folder must not become legacy behaviour

### ISSUE THREAD

**Issue by janhancic** [BladeRunnerJS/brjs#1408]
I'm test upgrading our app to CT that uses BRJS RC1.

I get this warning in the console:

```
BRJS now uses a folder named 'brjs-apps' for the location of your apps but the directory 'd:\solutions\st\sdk' contains both 'brjs-apps' and 'apps' folders. 'd:\solutions\st\sdk/apps' will be used for the location of apps but this legacy behaviour may be removed so you should move all existing apps into the 'd:\solutions\st\sdk/brjs-apps' directory.
```

First issue is that the paths reported are wrong. Both folders are in the root of the repository, not in the SDK folder.

The second issue is that making `apps` legacy is unacceptable. We will not move our apps to a different folder as that will cause all sort of different problems.

**Comment by andyberry88** [comment]
We decided to rename the directory when we looked at allowing apps to be separate from the SDK. It was done so we could be 100% sure that the directory was for brjs apps and not another directory that happened to be called `apps`. 

Now we're made a change to default to the working dir as the apps location (#1386) we probably don't need this `brjs-apps` change and we can revert back to use `apps` as the expected location for apps, otherwise default to the current working dir.

@dchambers @ioanalianabalas thoughts?

**Comment by dchambers** [comment]
@andyberry88, if you are sure this is no longer necessary than of course it would be much better to only have one location rather than two, and given that we've only started using this new location in a _release-candidate_ than I'm happy to just stop supporting `brjs-apps` completely.

**Comment by ioanalianabalas** [comment]
I think that given we are currently using the working directory, we could make that change, definitely. Will we now be checking the existence of an `apps` directory in the current working directory or parallel to the `sdk`? What is the order of our checks?

1. Look for `apps` in current directory
2. Look for `apps` next to the sdk

**Comment by andyberry88** [comment]
I'd suggest we do the following:
- recursively traverse up the directory tree checking for `app.conf` or the `apps` directory (we already do this)
- if nothing is found default to the current working directory for the apps location (this is added in #1386)

The change would be to remove the additional check for `brjs-apps` and rename our of `brjs-apps` directory back to `apps`. The PR https://github.com/BladeRunnerJS/brjs-site/pull/97 wouldn't now be needed either.

**Comment by dchambers** [comment]
Sounds good :+1:, @ioanalianabalas, are you happy with this?

**Comment by ioanalianabalas** [comment]
Yes, should work fine :+1:

**Comment by ioanalianabalas** [comment]
I will start working on the modifications of this one, as #1414 depends on these changes.

**Comment by ioanalianabalas** [comment]
Fixed by #1417.

**Comment by andyberry88** [comment]
:+1: looks good
